tests/integration/docker_images/popular-containers-images.bats

```diff
@@ -0,0 +1,405 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# Tests for the most popular images from docker hub
+
+setup() {
+	load common
+	cleanDockerPs
+	runtimeDocker
+}
+
+@test "run hello world in java in an openjdk container" {
+	$DOCKER_EXE run --rm -i openjdk bash -c "echo 'public class HW{public static void main(String[]a){System.out.println(\"HelloWorld\");}}' > HW.java && javac HW.java && java HW"
+}
+
+@test "check kong configuration file is valid" {
```

Comment:
For consistency, it would be good if all the test names had the image name in a fix place. You've mostly done that but this test name doesn't end with `$image_name container`.

Aside: I'd personally prefer this format:

```
@test "$image_name container: $description_of_test_being_run"
```
but this isn't a blocker for merging this PR :-)